language/move-lang/src/parser/syntax.rs

```diff
@@ -588,34 +582,19 @@ fn parse_byte_string(tokens: &mut Lexer) -> Result<Value_, Error> {
     Ok(value_)
 }
 
-// Create an error for an integer literal that is too big to fit in its type.
-// This assumes that the literal is the current token.
-fn num_too_big_error(tokens: &mut Lexer, type_description: &str) -> Error {
-    let start_loc = tokens.start_loc();
-    let end_loc = start_loc + tokens.content().len();
-    let loc = make_loc(tokens.file_name(), start_loc, end_loc);
-    vec![(
-        loc,
-        format!(
-            "Invalid number literal. The given literal is too large to fit into the {}",
-            type_description
-        ),
-    )]
-}
-
 // Parse a value:
 //      Value =
-//          <Address>
+//          "@" <Number>
```

Comment:
This should still refer to `<Address>` (along with "@"). We've still got a `parse_address` function that does something different than just `parse_number`, so effectively there's still an `<Address>` non-terminal in the grammar.